trade-finance-manager-api/src/v1/controllers/acbs.controller.js

```diff
@@ -36,8 +36,8 @@ const clearACBSLog = async () => {
 
 const createACBS = async (deal) => {
   // Add bank's full details so we can reference partyUrn in function
-  const bankId = deal.dealSnapshot.bankId
-    ? deal.dealSnapshot.bankId
+  const bankId = deal.dealSnapshot.bank
+    ? deal.dealSnapshot.bank.id
     : deal.dealSnapshot.details.owningBank.id;
```

Comment:
`owingBank.id` will it be a separate PR where it will appear under `bank` object?

Reply:
Yes will be seperate PR